### PR TITLE
build(JSAM-17): Login frontend

### PR DESCRIPTION
**Pull request by Cola-LIAN**
> **Finish JIRA Story: [JSAM-17](https://jira.greenfox.academy/browse/JSAM-17).**

### Finish:
- Create a basic `Login` page
- Stylish the page with CSS styles according to the mockup
- Create `Select`, `InputText` and `Link` as shared components
- Fill the "Click here"&"Sign in to an existing account" link button with navigation
- Create test file

Required|Fulfilled
----|----
image|image

### CODE REVIEW

File: JSA-Mawsitsit-Frontend/components/SharedUnits/Select.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+// Local Variables
+const styles = StyleSheet.create(loginStyles);
+
+const Select = (props) => {
+  const { selectItem } = props;
+  return (
+    <View style={styles.selectStyle}>
+      <Text style={styles.selectText}>{selectItem}</Text>
+      <Switch
+        // thumbColor={[status? '#c1d045':'#fff')]}
```
Comment (matecserven) [inline JSA-Mawsitsit-Frontend/components/SharedUnits/Select.js]: dont leave comments in

File: JSA-Mawsitsit-Frontend/components/Login/styles.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+const loginStyles = {
```
Comment (matecserven) [inline JSA-Mawsitsit-Frontend/components/Login/styles.js]: create the stylesheet here, and you don't have to create it 4 separate time
also stylesheet should go to the src/styles folder

File: JSA-Mawsitsit-Frontend/components/SharedUnits/Select.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+// External Dependencies
```
Comment (matecserven) [inline JSA-Mawsitsit-Frontend/components/SharedUnits/Select.js]: These sharedUnits are used by Login screen, place them in the Login folder

**Comment by matecserven** [comment]
rebase to master as well

**Comment by matecserven** [comment]
also please create tests for the components

File: JSA-Mawsitsit-Frontend/src/components/Login/Footer.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+} = LoginStyle;
+
+
+// Component Definition
+const Footer = () => (
+  <View style={footerStyle}>
+    <View style={footerFrameStyle}>
+      <Text style={footerTextStyle}>Don&apos;t have an account?</Text>
+      <View style={footerFrameInsideStyle}>
+        <Link
```
Comment (matecserven) [inline JSA-Mawsitsit-Frontend/src/components/Login/Footer.js]: Navigation is implemented, this component can send the user to the register screen

File: JSA-Mawsitsit-Frontend/src/components/Login/Input.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  };
+
+  const handleSelect = (name, value) => {
+    // const validateString = `validation.${name}`;
+    // const validateWord = Boolean(validateString);
+    const validator = validation[name];
+
+    // console.log(validator);
+    console.log(inputText.password !== 0);
+    console.log(inputText);
```
Comment (matecserven) [inline JSA-Mawsitsit-Frontend/src/components/Login/Input.js]: remove console.logs

File: JSA-Mawsitsit-Frontend/src/components/Login/Input.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+  const handleChange = (name, value) => {
+    setInputText({
+      ...inputText,
+      [name]: value,
+    });
+  };
+
+  const handleSelect = (name, value) => {
+    // const validateString = `validation.${name}`;
```
Comment (matecserven) [inline JSA-Mawsitsit-Frontend/src/components/Login/Input.js]: remove comments

File: JSA-Mawsitsit-Frontend/src/navigation/AppNavigation.js [hunk 7]
```diff
@@ -8,11 +8,13 @@ import {
...
 // Internal Dependencies
-import PersonalDetails from '../PersonalDetails/PersonalDetails';
-import Register from '../Register/Register';
+import PersonalDetails from '../components/PersonalDetails/PersonalDetails';
+import LoginScreen from '../screens/LoginScreen';
+import RegisterScreen from '../screens/RegisterScreen';
 
 const AuthStack = createStackNavigator({
-  SignUp: Register,
+  SignIn: LoginScreen,
```
Comment (matecserven) [inline JSA-Mawsitsit-Frontend/src/navigation/AppNavigation.js]:
```suggestion
  Login: LoginScreen,
```

File: JSA-Mawsitsit-Frontend/src/navigation/AppNavigation.js [hunk 8]
```diff
@@ -8,11 +8,13 @@ import {
...
+  SignUp: RegisterScreen,
```
Comment (matecserven) [inline JSA-Mawsitsit-Frontend/src/navigation/AppNavigation.js]:
```suggestion
  Register: RegisterScreen,
```

**Comment by matecserven** [comment]
solve merge conflict please

File: JSA-Mawsitsit-Frontend/App.js [hunk 9]
```diff
@@ -1,8 +1,10 @@
 // External Dependencies
 import React from 'react';
+import {
+} from 'react-native';
```
Comment (binjiezhang0626) [inline JSA-Mawsitsit-Frontend/App.js]: useless import

**Pull request merged**